<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-19T10:19:00 CLST" name="Automation_Concepts_Project" finished-at="2025-03-19T10:19:07 CLST" duration-ms="7806">
    <groups>
    </groups>
    <test started-at="2025-03-19T10:19:00 CLST" name="TestBase" finished-at="2025-03-19T10:19:07 CLST" duration-ms="7806">
      <class name="cl.evaluacion.qa.tests.TestTrello">
        <test-method is-config="true" signature="configExtentReports()[pri:0, instance:cl.evaluacion.qa.tests.TestTrello@2ea6137]" started-at="2025-03-19T10:18:59 CLST" name="configExtentReports" finished-at="2025-03-19T10:19:00 CLST" duration-ms="104" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configExtentReports -->
        <test-method is-config="true" signature="configSelenium(java.lang.String)[pri:0, instance:cl.evaluacion.qa.tests.TestTrello@2ea6137]" started-at="2025-03-19T10:19:00 CLST" name="configSelenium" finished-at="2025-03-19T10:19:04 CLST" duration-ms="4217" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://the-internet.herokuapp.com/login]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configSelenium -->
        <test-method signature="inicioSesion()[pri:0, instance:cl.evaluacion.qa.tests.TestTrello@2ea6137]" started-at="2025-03-19T10:19:04 CLST" name="inicioSesion" finished-at="2025-03-19T10:19:07 CLST" duration-ms="3357" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Your username is invalid!] but found [Your username is invalid!
×]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Your username is invalid!] but found [Your username is invalid!
×]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at cl.evaluacion.qa.pages.LoginPage.assertIngreso(LoginPage.java:51)
at cl.evaluacion.qa.tests.TestTrello.inicioSesion(TestTrello.java:80)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- inicioSesion -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:cl.evaluacion.qa.tests.TestTrello@2ea6137]" started-at="2025-03-19T10:19:07 CLST" name="afterMethod" finished-at="2025-03-19T10:19:07 CLST" duration-ms="215" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=inicioSesion status=FAILURE method=TestTrello.inicioSesion()[pri:0, instance:cl.evaluacion.qa.tests.TestTrello@2ea6137] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="closeExtentReports()[pri:0, instance:cl.evaluacion.qa.tests.TestTrello@2ea6137]" started-at="2025-03-19T10:19:07 CLST" name="closeExtentReports" finished-at="2025-03-19T10:19:08 CLST" duration-ms="158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeExtentReports -->
      </class> <!-- cl.evaluacion.qa.tests.TestTrello -->
    </test> <!-- TestBase -->
  </suite> <!-- Automation_Concepts_Project -->
</testng-results>
